resources/META-INF/plugin.xml

```diff
@@ -121,7 +121,7 @@
     <typeDeclarationProvider implementation="com.goide.editor.GoTypeDeclarationProvider"/>
     <codeInsight.parameterInfo language="go" implementationClass="com.goide.editor.GoParameterInfoHandler"/>
     <lang.smartEnterProcessor language="go" implementationClass="com.goide.editor.smart.GoSmartEnterProcessor"/>
-    <highlightUsagesHandlerFactory implementation="com.goide.highlighting.GoHighlightExitPointsHandlerFactory"/>
+    <highlightUsagesHandlerFactory implementation="com.goide.highlighting.exitpoint.GoHighlightExitPointsHandlerFactory"/>
```

Comment:
Oops, missed that. I believe it's more about `usages` than `exitpoints`. Maybe we'll highlight something more in future